plugin/src/main/java/com/denizenscript/denizen/objects/properties/material/MaterialDistance.java

```diff
@@ -79,41 +85,16 @@ public boolean isLeaves() {
         return material.getModernData() instanceof Leaves;
     }
 
-    @Override
-    public String getPropertyString() {
-        return String.valueOf(getDistance());
-    }
-
-    @Override
-    public String getPropertyId() {
-        return "distance";
+    public static MaterialDistance getFrom(MaterialTag _material) {
+        if (!describes(_material)) {
+            return null;
+        }
+        else {
+            return new MaterialDistance(_material);
+        }

```

Comment:
There's shouldn't be any need to keep `getFrom` - if anything is using this property directly, can use `describes` & add a constructor that directs to `super`.
In general should ideally remove the older methods when updating a property, any `is/getX` methods can just be `instanceof` pattern matching, any `get/setDistance` sorta methods can usually just be put directly in `get/setPropertyValue`, etc..
It's not an absolute _requirement_, but it'd be nicer to have the properties fully match the modern format.